packages/connection-form/src/components/advanced-connection-options.tsx

```diff
@@ -26,6 +26,7 @@ const disabledOverlayStyles = css({
 
 const connectionTabsContainer = css({
   position: 'relative',
+  paddingLeft: spacing[2],
```

Comment:
Without this, a bit of the tab content was cut out in the modal version. I couldn't figure out why, so this is a quick fix - the extra padding is barely visible in the legacy form.